addons/mail/static/src/components/activity_box/activity_box.scss

```diff
@@ -14,6 +14,10 @@
     padding: map-get($spacers, 0) map-get($spacers, 2);
 }
 
+.o_ActivityBox_titleBadges {
+    margin-inline-end: map-get($spacers, 3);
```

Comment:
It's strange that `o_ActivityBox_titleText` is using padding, which serves as left spacer and spacer between the 2 elements, but for the right spacer using a margin on the badges.

I think this should be uniformized.
Probably margin is more suitable since the goal is to add space between the elements, not make the elements themselves bigger?

Reply:
It's the same for me, I understand that margin makes more sense, changed it :)